cli/tests/conftest.py

```diff
@@ -126,7 +127,7 @@ def _make_message(project_name: str, accession_number: str, mrn: str) -> Message
         mrn=mrn,
         study_date=STUDY_DATE,
         procedure_occurrence_id=1,
-        extract_generated_timestamp=datetime.datetime.now(tz=datetime.UTC),
+        extract_generated_timestamp=datetime.datetime.now(timezone.utc),
```

Comment:
if there's a bug here it's probably best to open a separate issue / PR to fix it, just to keep it distinct from the doc improvements. Also, it looks like `timezone.UCT` was [added in Python 3.11](https://docs.python.org/3/library/datetime.html#datetime.UTC). As the tests run in Python 3.11, perhaps we should suggest upgrading from 3.10?

Reply:
:heavy_check_mark: e6840d53c7fce5157e9cfd4ceac48548cde16e26